src/HotChocolate/Core/src/Types/Types/Descriptors/ObjectTypeDescriptor.cs

```diff
@@ -120,21 +120,21 @@ internal void InferFieldsFromFieldBindingType()
         var fields = TypeMemHelper.RentObjectFieldDefinitionMap();
         var handledMembers = TypeMemHelper.RentMemberSet();
 
-        InferFieldsFromFieldBindingType(fields, handledMembers);
+        InferFieldsFromFieldBindingType(fields, handledMembers, false);
 
         TypeMemHelper.Return(fields);
         TypeMemHelper.Return(handledMembers);
     }
 
-    protected void InferFieldsFromFieldBindingType(
+    internal void InferFieldsFromFieldBindingType(
```

Comment:
why do we need this internal?

Reply:
it feels very internal

Reply:
because i did too much java at uni and though `private protected` is a java only thing 😄